### PR TITLE
Pass font variation with font-family

### PR DESCRIPTION
**Pull request by ramghaju**
Webfont loader with Monotype module always shows "n4" as font variation description. Monotype users can enable family grouping on their project after adding fonts on the project. When this project is used with google webfont loader, it always shows "n4" as font variation description. So, it is firing single font event for different variation of same font-family.
for more detail: https://github.com/typekit/webfontloader/issues/272

I have updated logic to pass font variation along with font-family.

### CODE REVIEW

**Comment by bramstein** [comment]
This looks good. Minor nitpick: could you add spaces at appropriate places (e.g. after a comma, before and after an equals sign, etc.).

File: src/modules/monotype.js [hunk 1]
```diff
@@ -58,11 +58,20 @@ goog.scope(function() {
...
 
         if (mti_fnts) {
           for (var i = 0; i < mti_fnts.length; i++) {
-            fonts.push(new Font(mti_fnts[i]["fontfamily"]));
+            var fnt = mti_fnts[i]["fontfamily"];
+            
+            //Check if font-style and font-weight is available
+            if (mti_fnts[i]["fontStyle"] != undefined && mti_fnts[i]["fontWeight"] != undefined) {
+              fntVariation = mti_fnts[i]["fontStyle"] + mti_fnts[i]["fontWeight"];
+              fonts.push(new Font(fnt,fntVariation));
```
Comment (bramstein) [inline src/modules/monotype.js]: Missed one here :)

**Comment by ramghaju** [comment]
I have added missing space. 

A quick question, how can we know these changes are reflected on production environment?

Thanks,
Ram

**Comment by bramstein** [comment]
I'll do a release right now, and then within a couple weeks the Google CDN will pick it up. Unfortunately there isn't a more detailed schedule as I don't have any control over Google's release schedule.

**Comment by ramghaju** [comment]
Thanks @bramstein

**Pull request merged**